content/en/docs/v3.5/op-guide/data_corruption.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Data Corruption
+weight: 5000
+description: etcd data corruption and recovery
+---
+
+etcd has built in automated data corruption detection to prevent member state from diverging.
+
+## Enabling data corruption detection
+
+Data corruption detection can be done in two ways:
+* Initial check, enabled with `--experimental-initial-corrupt-check` flag.
+* Periodic check, enabled with `--experimental-corrupt-check-time` flag.
+
+Initial check will be executed during bootstrap of etcd member.
+Member will compare it's persistent state vs other members and exit if there is a mismatch.
+
+Periodic check will be executed by the cluster leader in a cluster that is already running.
+Leader will compare it's persistent state vs other members and raise a CORRUPT ALARM if there is a mismatch.
+Period of checks is configured using format: `1m` - every minute, `1h` - evey hour.
+Recommended period is a couple of hours as there is a high performance cost.
```

Comment:
as the checksum is computed by active scanning entire etcd content at given revision.

Reply:
Done